source/includes/administration/_discounts.md

```diff
@@ -0,0 +1,144 @@
+`discountScope`<br/>*enum* | The scope of the discount. It could be either "ALL_PRODUCTS", "CATEGORIES" or "PRODUCTS".
+`isDeactivated`<br/>*boolean* | Whether or not the discount is deactivated. Defaults to false.
+`discountedCategories`<br/>*Map[UUID, BigDecimal]* | A mapping between category IDs and discount values. All pricing products within the categories specified will have the discount value applied to them.
+`name`<br/>*Map[String, String]* | The name translations of the discount.
+`appliedPricing`<br/>*Object* | The object representing the applied pricing owning the discount.
+`appliedPricing.id`<br/>*UUID* | The UUID of the applied pricing.
+`applyToNewCustomersOnly`<br/>*boolean* | If true, the discount will only be applied to organizations created after the discount start date.
+`startDate`<br/>*date* | The start date of the discount.
+`cutoffDate`<br/>*date* | The date on which the discount will no longer be available to customers who have not already received it. If not provided, the discount will always be available after the start date.
+`status`<br/>*enum* | The status of the discount. Possible values are : UPCOMING, CURRENT, ENDED.
+
+
+<!-------------------- CREATE DISCOUNTS -------------------->
+### Create discounts
+
+`POST /applied_pricings/:applied_pricing_id/discounts`
+
+Creates a new discount 
+
+```shell
+# Creates a new discount
+curl -X POST "https://cloudmc_endpoint/rest/applied_pricings/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts" \
+   -H "MC-Api-Key: your_api_key"
+```
+> Request body example:
+
+```json
+{
+  "name": {
+    "en": "Summer Discount",
+    "fr": "Réduction Estival"
+  },
+  "startDate": "2021-07-23T00:00:00.000Z",
+  "type": "PERCENTAGE",
+  "durationMonths": 4,
+  "discountedProducts": {},
+  "discountedCategories": {
+    "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+    "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+  },
+  "discountScope": "CATEGORIES", 
+  "cutoffDate": "2021-07-24T00:00:00.000Z",
+  "applyToNewCustomersOnly": false,
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+      "applyToNewCustomersOnly": false,
+      "discountedProducts": {},
+      "durationMonths": 4,
+      "type": "PERCENTAGE",
+      "discountScope": "CATEGORIES",
+      "discountedCategories": {
+        "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+        "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+      },  
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "appliedPricing": {
+        "id": "efd32752-c6f2-45cf-b494-cc6be8a45845"
+      },
+      "startDate": "2021-07-23T00:00:00.000Z",
+      "cutoffDate": "2021-07-24T00:00:00.000Z",
+  }
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*Map[String, String]* | The name translations of the discount.
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
```

Comment:
It can either be "PERCENTAGE" or "CREDIT".

Reply:
035b60e